src/integrations/prefect-databricks/tests/test_flows.py

```diff
@@ -25,8 +25,14 @@ async def async_handler(result, state):
 
 
 @pytest.fixture
-def run_now_mocks(respx_mock):
-    respx_mock.post(
+def respx_mock_with_pass_through(respx_mock):
+    respx_mock.route(host="127.0.0.1").pass_through()
```

Comment:
This passthrough ensures that calls from the prefect client make it to the local server.